frame/nomination-pools/src/lib.rs

```diff
@@ -1782,15 +1953,12 @@ impl<T: Config> Pallet<T> {
 	///
 	/// Returns the payout amount.
 	fn calculate_delegator_payout(
+		delegator: &mut Delegator<T>,
 		bonded_pool: &mut BondedPool<T>,
 		reward_pool: &mut RewardPool<T>,
-		delegator: &mut Delegator<T>,
```

Comment:
Just out of curiosity, why move the delegator param?

Reply:
to make it more aligned with the argument order of the rest of the calls.